BASE/Test/ServerTelemetryChannel.Test/TelemetryChannel.Tests/TelemetryChannel.Tests.csproj

```diff
@@ -22,6 +22,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
```suggestion
  <ItemGroup Condition="'$(TargetFramework)' != 'net452'">
```

Reply:
```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'NETCORE'"> 
```

woudl this work!

Reply:
I haven't found a way to use wildcards in the csproj.
The `NETCOREAPP` key word is only available as a preprocessor. 
link: https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries#how-to-multitarget

At line 24 specifically, this wouldn't work because these dependencies are for netcoreapp 2.1 only. These aren't shared with netcoreapp 3.1.